src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1715,6 +1724,81 @@ private void SetPeriodicEvaluation(WorkspaceModel ws)
             return true;
         }
 
+        // Reload's the dummy nodes and serializes the new node information once the package resolves it. 
```

Comment:
this comment is a bit hard to make sense of.

Reply:
Changed the description.